pkg/api/v1/resource/helpers.go

```diff
@@ -248,6 +248,61 @@ func max(a v1.ResourceList, b v1.ResourceList) v1.ResourceList {
 	return result
 }
 
+// MinRegularContainerResourceList returns the minimum resource requests for each named resource
+// across all regular containers in pod.Spec.Containers.
+// It does not include initContainers (both restartable and non-restartable).
+func MinRegularContainerResourceList(pod v1.Pod, opts PodResourcesOptions) v1.ResourceList {
```

Comment:
I don't see a reason to make it a part of an API. Keep it as a local helper function under the kubelet/qos

Reply:
done